### PR TITLE
Fix addition of elements to a bash array

### PR DESCRIPTION
**Pull request by matejak**
See e.g. https://opensource.com/article/18/5/you-dont-know-bash-intro-bash-arrays

### CODE REVIEW

File: shared/templates/pam_options/bash.template [hunk 1]
```diff
@@ -7,26 +7,26 @@
...
 declare -a ARGS=()
-declare -a NEW_ARS=()
+declare -a NEW_ARGS=()
 
 {{% for arg in ARGUMENTS -%}}
 {{% if arg['variable']|length -%}}
     {{{ bash_instantiate_variables("var_password_pam_" + arg['variable']) -}}}
-VALUES+=${{{ "var_password_pam_" + arg['variable'] }}}
-VALUE_NAMES+={{{ arg['variable'] }}}
+VALUES+=("${{{ "var_password_pam_" + arg['variable'] }}}")
```
Comment (jan-cerny) [inline shared/templates/pam_options/bash.template:15]: I have discovered another problem. The VALUES is glued after NEW_ARGS. Build the SLE 15 content and check build/sle15/fixes/bash/accounts_password_pam_pwhistory_remember.sh:

```

var_password_pam_remember='(bash-populate var_password_pam_remember)'
VALUES+=("$var_password_pam_remember")
VALUE_NAMES+=("remember")
ARGS+=("")
NEW_ARGS+=("")VALUES+=("")
VALUE_NAMES+=("")
ARGS+=("use_authtok")
NEW_ARGS+=("use_authtok")
```

**Comment by jan-cerny** [comment]
I wonder how did you discover this issue. Were there any tests failing or some tools reporting something? Or did you just accidentally see it?

**Comment by matejak** [comment]
> I wonder how did you discover this issue. Were there any tests failing or some tools reporting something? Or did you just accidentally see it?

It got discovered by shellcheck, and it seemed to be a good candidate to separate to another PR, as the template was really broken.

File: shared/templates/pam_options/bash.template (hunk 1 above)
Comment (jan-cerny) [reply shared/templates/pam_options/bash.template:15]: it still happens, even after your push-force

Comment (matejak) [reply shared/templates/pam_options/bash.template:15]: thanks, now it should be OK.

**Review by jan-cerny — APPROVED** [review]

**Pull request merged**